examples/basic/android/app/build.gradle

```diff
@@ -127,9 +131,10 @@ android {
 }
 
 dependencies {
-    compile "com.android.support:appcompat-v7:25.2.0"
-    compile "com.facebook.react:react-native:+"  // From node_modules
-    compile project(':react-native-video')
+    implementation project(':react-native-video')
+    compile "com.android.support:appcompat-v7:28.0.0"
+    implementation "com.facebook.react:react-native:+"  // From node_modules
+    // implementation project(':react-native-video')
```

Comment:
huh, I meant that this line can be removed 😅 	
```suggestion
```

Reply:
oh, my mistake  😅

Reply:
new commit change the dead code and remove the repeated line